test/cartesian/YAxis.spec.tsx

```diff
@@ -72,20 +75,19 @@ describe('<YAxis />', () => {
     expect(ticks[1].getAttribute('y')).toBe('297.5');
   });
 
-  each([[[0, 'dataMax + 100']], [[0, 'dataMax - 100']], [['auto', 'auto']]]).it(
-    'Should render 0 ticks when domain={%s} and dataKey is "noExist" ',
-    domain => {
-      render(
-        <AreaChart width={600} height={400} data={data}>
-          <YAxis stroke="#ff7300" domain={domain} />
-          <Area dataKey="noExist" stroke="#ff7300" fill="#ff7300" />
-        </AreaChart>,
-      );
-      const ticks = document.querySelectorAll('text');
-
-      expect(ticks).toHaveLength(0);
-    },
-  );
+  const cases2: [AxisDomain][] = [[[0, 'dataMax + 100']], [[0, 'dataMax - 100']], [['auto', 'auto']]];
```

Comment:
If we do not want to use good names, we could also just inline the object as well.

Reply:
If I inline then I cannot have explicit type definition. I could use a sea of `as const` and rely on typescript to infer what I need which I didn't want to do. Or I could use `as` which I don't want since it's just `any` in a different coat. Or I could use `any` which I don't want.

I found a better name instead!